<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPUConfig</class>
 <widget class="QMainWindow" name="CPUConfig">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPUConfig</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>CPU Controls</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>254</width>
          <height>130</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Min CPU Frequency:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDial" name="cpuMin">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>900000</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cpuMinBox">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>900000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Mhz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>251</width>
          <height>130</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Max CPU Frequency:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDial" name="cpuMax">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>900000</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="cpuMaxBox">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>900000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Mhz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="applyChanges">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply changes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resetButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>86</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QLabel" name="rootAccess">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>350</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ENABLED</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Root access:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CPU Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="CPUStatTable">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Average CPU Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Allowed Minimum CPU Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Allowed Maximum CPU Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Current minimum CPU Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Current maximum CPU Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Number of CPU cores</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>CPU Model Name</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
